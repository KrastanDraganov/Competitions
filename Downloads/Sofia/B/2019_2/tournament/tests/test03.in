10
7 1 2 5 6 7 8 9
4 5 6 7 9
6 1 5 6 7 8 9
9 0 1 2 4 5 6 7 8 9
8 0 1 2 5 6 7 8 9
2 7 9
3 5 7 9
1 9
5 1 5 6 7 9
0
