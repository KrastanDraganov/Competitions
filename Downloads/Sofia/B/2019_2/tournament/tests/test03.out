3 4 0 2 8 1 6 5 7 9 
